dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/EventManager.java

```diff
@@ -248,6 +248,7 @@ public ImportSummaries updateEvents( final List<Event> events, final WorkContext
             catch ( Exception e )
             {
                 handleFailure( workContext, importSummaries, events, IMPORT_ERROR_STRING, UPDATE );
+                log.error( "Failed to update events: " + e );
```

Comment:
Please use https://www.javadoc.io/doc/org.slf4j/slf4j-api/1.7.25/org/slf4j/Logger.html#error(java.lang.String,%20java.lang.Throwable) instead of string concatenating the exception 😄